### ISSUE TITLE
IPsec routed mode has changed, please update docs

### ISSUE THREAD

**Issue by ppbrown** [opnsense/core#5518]
This is from version 21.7.7, and is related to 

https://github.com/opnsense/core/issues/4029

Before that bug, the docs at https://docs.opnsense.org/manual/how-tos/ipsec-s2s-route.html
were mostly valid, except in odd cases.

Now they are incorrect in multiple places.

With certain other endpoints (ie: meraki) you cannot use the fake "tunnel network", like in the example of using 10.111.1.1 and 10.111.1.2
You HAVE to use the real IP addresses. (and if NATted, you need to use the public IP ones), otherwise auth fails
(at least for meraki routers)
This is true even if I set the "local Id" and "remote id" on the other side to match the fake 10.11.1.1 addrs.

Additionally, to get it working, I had to NOT set the "far gateway" checkbox, contradicting the current docs.

full details of my setup:

Meraki vMX version 15.44 in google cloud, obviously NATted. Mujltiple subnets behind it.
OPNsense 22.7.7 router behind home router and internet, also NATted. Single subnet behind it.

- [x ] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x ] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

Disclaimer: I have not tested the behavior if both sides are OPNsense. I'm reporting on my specific setup.
Maybe the docs still actually work when both sides are OPNsense. But even if they do... it will be easier for more people to use,  if you change them to not have the fake intermediate IP addresses.

**Comment by mimugmail** [comment]
Maybe this is only related to Meraki? I can't imagine tunnel IPs are involved there. 
For ID's you could also set FQDNs, no idea if Meraki supports this.

**Comment by ppbrown** [comment]
If I set anything other than the real IPs it fails auth. it seems pretty clear.
THAT BEING SAID....
even if that problem is limted to a subset of devices, when there's a choice between:

a) a configuration that is more complex, and works on most-but-not-all devices
vs
b) a configuration that is easy(er) to understand, and works on all devices

seems like it would be a pretty clear choice to migrate to option b ?

**Comment by mimugmail** [comment]
Can you post logs of both devices with the known working config when IPsec fails please?

**Comment by ppbrown** [comment]
you're not even interested in the documentation win?
you dont want to try out for yourself, if setting the actual IPs instead of fake IPs works for opnsense to opensense, and if so, to simplify the docs?

**Comment by mimugmail** [comment]
No offense, but I'm interested in your problem and the solution, not in updating the docs cause someone said something without posting anything.

**Comment by ppbrown** [comment]
I already posted the problem I had, AND the solution. in DETAIL. The first post of this ticket. Your claims that I "havent posted anything" are odd in the extreme.

I've already fixed my connectivity problem. I already found the solution.
In the interests of open source, I wanted to share the knowlege, so other customers would have an easier time.

I thought this product was backed by a company interested in good customer service. Good customer service begins with good documentation. Are you not interested in easier to follow documentation?

FYI, my company is evaluating which solution we are going to chose, for a 1,000+ site  network. That means a sale of 1,000+ licenses.

Up until now, my recommendation has been trending towards OPNsense business edition. But if this is the kind of attitude we have to deal with when we run into issues, I'm reconsidering alternatives.
